Cafe linking issue with glslang 15

### Current Behavior

```
[532/532] Linking CXX executable /home/kescher/.cache/paru/clone/cemu/src/Cemu/bin/Cemu_release
/usr/bin/ld: /tmp/lto-llvm-b58442.o: in function `RendererShaderVk::CompileInternal(bool)':
/usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:314:(.text._ZN16RendererShaderVk15CompileInternalEb+0x131): undefined reference to `glslang::TShader::TShader(EShLanguage)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:316:(.text._ZN16RendererShaderVk15CompileInternalEb+0x154): undefined reference to `glslang::TShader::setStrings(char const* const*, int)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:334:(.text._ZN16RendererShaderVk15CompileInternalEb+0x22e): undefined reference to `glslang::TShader::preprocess(TBuiltInResource const*, int, EProfile, bool, bool, EShMessages, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, glslang::TShader::Includer&)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:348:(.text._ZN16RendererShaderVk15CompileInternalEb+0x299): undefined reference to `glslang::TShader::setStrings(char const* const*, int)'
/usr/bin/ld: /tmp/lto-llvm-b58442.o: in function `glslang::TShader::parse(TBuiltInResource const*, int, EProfile, bool, bool, EShMessages)':
/usr/include/glslang/Public/ShaderLang.h:673:(.text._ZN16RendererShaderVk15CompileInternalEb+0x2ea): undefined reference to `glslang::TShader::parse(TBuiltInResource const*, int, EProfile, bool, bool, EShMessages, glslang::TShader::Includer&)'
/usr/bin/ld: /tmp/lto-llvm-b58442.o: in function `RendererShaderVk::CompileInternal(bool)':
/usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:358:(.text._ZN16RendererShaderVk15CompileInternalEb+0x303): undefined reference to `glslang::TProgram::TProgram()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:361:(.text._ZN16RendererShaderVk15CompileInternalEb+0x348): undefined reference to `glslang::TProgram::link(EShMessages)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:369:(.text._ZN16RendererShaderVk15CompileInternalEb+0x361): undefined reference to `glslang::TProgram::mapIO(glslang::TIoMapResolver*, glslang::TIoMapper*)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:388:(.text._ZN16RendererShaderVk15CompileInternalEb+0x412): undefined reference to `glslang::GlslangToSpv(glslang::TIntermediate const&, std::vector<unsigned int, std::allocator<unsigned int> >&, spv::SpvBuildLogger*, glslang::SpvOptions*)'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:336:(.text._ZN16RendererShaderVk15CompileInternalEb+0x4f2): undefined reference to `glslang::TShader::getInfoLog()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:351:(.text._ZN16RendererShaderVk15CompileInternalEb+0x5ca): undefined reference to `glslang::TShader::getInfoLog()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:363:(.text._ZN16RendererShaderVk15CompileInternalEb+0x6ae): undefined reference to `glslang::TProgram::getInfoLog()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:371:(.text._ZN16RendererShaderVk15CompileInternalEb+0x78e): undefined reference to `glslang::TProgram::getInfoLog()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:410:(.text._ZN16RendererShaderVk15CompileInternalEb+0x92d): undefined reference to `glslang::TShader::~TShader()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:410:(.text._ZN16RendererShaderVk15CompileInternalEb+0xa09): undefined reference to `glslang::TProgram::~TProgram()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:410:(.text._ZN16RendererShaderVk15CompileInternalEb+0xb83): undefined reference to `glslang::TProgram::~TProgram()'
/usr/bin/ld: /usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.cpp:410:(.text._ZN16RendererShaderVk15CompileInternalEb+0xbc7): undefined reference to `glslang::TShader::~TShader()'
/usr/bin/ld: /tmp/lto-llvm-a0818c.o: in function `VulkanRenderer::VulkanRenderer()':
/usr/src/debug/cemu/Cemu/src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp:326:(.text._ZN14VulkanRendererC1Ev+0x371): undefined reference to `glslang::InitializeProcess()'
```

### Expected Behavior

A successful link

### Steps to Reproduce

1. Don't use vcpkg
2. Use glslang 15.0.0 (or, in the future, higher)

### System Info (Optional)

OS: Arch Linux

### Emulation Settings (Optional)

_No response_

### Logs (Optional)

_No response_

--- a/src/Cafe/CMakeLists.txt
+++ b/src/Cafe/CMakeLists.txt
@@ -532,6 +532,12 @@ set_property(TARGET CemuCafe PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CON
 
 target_include_directories(CemuCafe PUBLIC "../")
 
+if (glslang_VERSION VERSION_LESS "15.0.0")
+	set(glslang_target "glslang::SPIRV")
+else()
+	set(glslang_target "glslang")
+endif()
+
 target_link_libraries(CemuCafe PRIVATE
 	CemuAsm
 	CemuAudio
@@ -547,7 +553,7 @@ target_link_libraries(CemuCafe PRIVATE
 	Boost::nowide
 	CURL::libcurl
 	fmt::fmt
-	glslang::SPIRV
+	${glslang_target}
 	ih264d
 	OpenSSL::Crypto
 	OpenSSL::SSL

